### ISSUE TITLE
Rebundle Hello Browser on source file save

### ISSUE THREAD

**Issue by epiro-ps** [ArdanaLabs/dUSD#228]
We currently have to stop and start the nix run for hello browser every time via `nix run .#serve-hello-world-browser` when we want to recompile and confirm a change is properly appearing in the browser. It becomes costly over time as the developer has to switch contexts from the editor to the terminal to the browser when they want to see an update. They also have to wait for the terminal to indicate that the compilation is complete and the web server is up and refresh the browser. 

# Acceptance Criteria

As a developer, I want to run `nix run .#offchain:hello-world-browser:serve:dev` where the following is true:

When I make a change to a file within the [hello browser app](https://github.com/ArdanaLabs/dUSD/tree/master/offchain/hello-world-browser app directory), I expect the following behaviors:
- [x] The pursescript recompiles if changes where made to `*.purs` files.
- [x] The browser app is rebundled.

The following is nice to have.
- [ ] Once the rebundling is complete, the browser reloads automatically.
